artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/federation/AMQPFederation.java

```diff
@@ -77,6 +77,10 @@ public abstract class AMQPFederation implements FederationInternal {
    protected boolean started;
    protected volatile boolean connected;
 
+   public boolean isLargeMessageSync() {
```

Comment:
It isn't really necessary to define an accessor here as there are already accessors in the Session and Connection SPI bits.